easybuild/easyconfigs/n/NEST/NEST-2.12.0-foss-2015b.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'NEST'
+version = '2.12.0'
+
+homepage = 'http://www.nest-simulator.org/'
+description = """NEST is a simulator for spiking neural network models
+ that focuses on the dynamics, size and structure of neural systems rather
+ than on the exact morphology of individual neurons."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = ['nest-%(version)s.tar.gz']
+source_urls = ['https://github.com/nest/nest-simulator/releases/download/v%(version)s/']
+
+dependencies = [
+    ('Python', '2.7.9'),
+    ('libreadline', '6.3'),
+    ('GSL', '2.1'),
+    ('libtool', '2.4.6', '-GNU-4.9.3-2.25', True),
+]
+
+configopts = '-DREADLINE_ROOT_DIR=$EBROOTLIBREADLINE'
+
+builddependencies = [
+    ('CMake', '3.4.1'),
+    ('Doxygen', '1.8.10'),
+]
+
+modextrapaths = {
+    'PYTHONPATH': ['lib/python2.7/site-packages'],
```

Comment:
please don't hardcode the Python version, use the `%(pyshortver)s` template instead (see http://easybuild.readthedocs.io/en/latest/version-specific/easyconfig_templates.html)